karma.conf.js

```diff
@@ -51,7 +51,7 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['PhantomJS'],
+    browsers: [/*'Chrome'*/, 'PhantomJS'],
```

Comment:
theres no reason to have it in there commented out, uncomment it or get rid of it.